index.html

```diff
@@ -0,0 +1,68 @@
+<!doctype html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <title>Friends of Homeless Animals</title>
+  <link href="http://fonts.googleapis.com/css?family=Roboto+Slab:400,300,700&subset=latin,greek" rel="stylesheet" type="text/css">
+  <link rel="stylesheet" href="css/reset.css">
+  <link rel="stylesheet" href="css/main-styling.css">
+</head>
+<body>
+
+  <header id="page-header">
+    <h1><a href="index.html">Friends of Homeless Animals</a></h1>
+    <nav>
+      <ul>
+        <li><strong><a href="index.html">Home</a></strong></li>
```

Comment:
Like I mentioned above, try using `font-weight:bold` in your CSS instead of the `<strong>` element.